src/Workspaces/MSBuild/Core/MSBuild/BuildHostProcessManager.cs

```diff
@@ -382,12 +409,9 @@ public BuildHostProcess(Process process, string pipeName, ILoggerFactory? logger
 
             _process.ErrorDataReceived += Process_ErrorDataReceived;
 
-            var pipeClient = NamedPipeUtil.CreateClient(".", pipeName, PipeDirection.InOut, PipeOptions.Asynchronous);
+            // Inlined NamedPipeUtil.CreateClient because it does not support netstandard tfm due to using runtime specific functionality.
```

Comment:
Which functionality was missing?

Reply:
Ok. Found the System.IO.Pipes.AccessControl package which makes the security side of this straightforward. 

@jaredpar Would there be any interest in fleshing out a NETSTANDARD implementation of NamedPipeUtil?